test/Swashbuckle.AspNetCore.SwaggerGen.Test/XmlComments/XmlCommentsExampleHelperTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+using Xunit;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen.Test
+{
+    public class XmlCommentsExampleHelperTests
+    {
+        private readonly SchemaRepository schemaRepository = new SchemaRepository();
+
+        [Fact]
+        public void Create_BuildsOpenApiArrayJson__When_NotStringTypeAndDataIsArray()
+        {
+            OpenApiSchema schema = new OpenApiSchema();
+
+            IOpenApiAny example = XmlCommentsExampleHelper.Create(
+                schemaRepository,
+                schema,
+                "[\"one\",\"two\",\"three\"]");
+
+
+            var output = (OpenApiArray)example;
+            Assert.Equal(3, output.Count);
+            Assert.Equal("one",   ((OpenApiString)output[0]).Value);
+            Assert.Equal("two",   ((OpenApiString)output[1]).Value);
+            Assert.Equal("three", ((OpenApiString)output[2]).Value);
```

Comment:
Here and in the other methods, assert that the example isn't null as is of the right type, rather than doing casts without checking first.

Reply:
done

Reply:
This isn't done - the casts are still present.

The asserts should be something like this:

```csharp
var item = Assert.IsType<OpenApiArray>(output[0]);
Assert.Equal("one", item.Value);
```

Reply:
I hope, that it is now as expected